python/cugraph-pyg/cugraph_pyg/data/cugraph_store.py

```diff
@@ -188,34 +189,54 @@ class EXPERIMENTAL__CuGraphStore:
     # TODO add an "expensive check" argument that ensures the graph store
     # and feature store are valid and compatible with PyG.
     def __init__(
-        self, F, G, num_nodes_dict, backend: str = "torch", multi_gpu: bool = False
+        self,
+        F: cugraph.gnn.FeatureStore,
+        G: Union[Dict[str, Tuple[Tensor]], Dict[str, int]],
+        num_nodes_dict: Dict[str, int],
+        backend: str = "torch",
+        multi_gpu: bool = False,
     ):
         """
-        Constructs a new CuGraphStore from the provided
-        arguments.
-        Parameters
-        ----------
-        F : cugraph.gnn.FeatureStore (Required)
-            The feature store containing this graph's features.
-            Typed lexicographic-ordered numbering convention
-            should match that of the graph.
-        G : dict[tuple[tensor]] (Required)
-            Dictionary of edge indices.
-            i.e. {
-                ('author', 'writes', 'paper'): [[0,1,2],[2,0,1]],
-                ('author', 'affiliated', 'institution'): [[0,1],[0,1]]
-            }
-            Note: the internal cugraph representation will use
-            offsetted vertex and edge ids.
-        num_nodes_dict : dict (Required)
-            A dictionary mapping each node type to the count of nodes
-            of that type in the graph.
-        backend : ('torch', 'cupy') (Optional, default = 'torch')
-            The backend that manages tensors (default = 'torch')
-            Should usually be 'torch' ('torch', 'cupy' supported).
-        multi_gpu : bool (Optional, default = False)
-            Whether the store should be backed by a multi-GPU graph.
-            Requires dask to have been set up.
+                Constructs a new CuGraphStore from the provided
+                arguments.
+                Parameters
+                ----------
+                F : cugraph.gnn.FeatureStore (Required)
+                    The feature store containing this graph's features.
+                    Typed lexicographic-ordered numbering convention
+                    should match that of the graph.
+
+                G : dict[str, tuple[tensor]] or dict[str, int] (Required)
+                    Dictionary of edge indices.
+                    Option 1 (graph in memory):
+                        Pass the edge indices
+                        i.e. {
+                            ('author', 'writes', 'paper'): [[0,1,2],[2,0,1]],
+                            ('author', 'affiliated', 'institution'): [[0,1],[0,1]]
+                        }
```

Comment:
Not a change request but i wonder if we should allow users to also add `dask_cudf` frames here .

Reply:
fixed